### PR TITLE
Fish Classification

### PR DESCRIPTION
**Pull request by TheDarkParalda**
# Pull Request for ML-Crate 💡

## Issue Title: Fish Classification

- **Info about the related issue (Aim of the project)** : Create a ML model which will classify and identify the fishes.
- **Name:** Evan Joshy Chittilapilly
- **GitHub ID:** TheDarkParalda
- **Idenitfy yourself:** Participant

Closes: #141

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made
* Importing Keras, Tensorflow, mathplotlib for the solution
* Creating the function that returns the VGG_16 model and ResNet model
* Importing the data, Splitting the dataset to Train and Test (Validation) data and normalising them.
* Training the model. (Uses Early Stopping for maximum accuracy and Adam as an optimiser)
* Plotting the accuracy and loss per epoch

## Type of change ☑️

What sort of change have you made:
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested? ⚙️

The training set was split to test and train set and was tested on the test set.

## Checklist: ☑️
- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Create a folder named as **Images** and put your plots there. Also showcase your plots in the `readme.md` file under the heading of **Model Visualization**.

Please do the needful ASAP.
@TheDarkParalda

**Comment by TheDarkParalda** [comment]
Changes done. Please review.

TheDarkParalda

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Everything looks good to me. Just remove the `bruh` file.🤣

**Comment by TheDarkParalda** [comment]
Done. lol 👍

**Review by abhisheks008 — APPROVED** [review]
Bruh moment OP!
Your PR is approved and ready to be merged.
@TheDarkParalda
